google/cloud/forseti/scanner/scanners/data_models/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Config Validator data model folder."""
```

Comment:
Since these data models are specific to Config Validator, how about putting it into the `config_validator_util` folder?  It feels it should not be directly under `scanners`
```
forseti-security/google/cloud/forseti/scanner/scanners/config_validator_util/data_models
```

Reply:
@blueandgold Have moved these files under the config_validator_util folder.